Assets/Scripts/Character/Enemies/Enemy.cs

```diff
@@ -48,17 +48,19 @@ private void Update()
             }
         }
 
-        private void OnCollisionEnter2D(Collision2D collision)
+        private void OnTriggerEnter2D(Collider2D collider)
         {
-            if(collision.collider.tag == Enums.Tags.HeroWeapon.ToString())
+            //Disabled for now. Not all enemies might take direct damage.
+            //BasicEnemy only takes damage when not stunned, so damage is handled in LocalCollision
+            /*if(collider.tag == Enums.Tags.HeroWeapon.ToString())
```

Comment:
Don't just comment out code like this.  If you need to move something to a lower level then only change the part you need.   The more proper way of doing this would be to add a new abstract TakeDamage(int damage) method to the Enemy class.  This would allow the logic to remain as is, while still letting you have the basic enemy only take damage when not in stun.